gslib/commands/cp.py

```diff
@@ -631,147 +615,141 @@
                  - Source size.
                  - Bytes transferred.
                  - MD5 hash.
-                 - UTC date and time transfer was started in ISO 8601 format.
-                 - UTC date and time transfer was completed in ISO 8601 format.
+                 - UTC date and transfer start time in ISO 8601 format.
+                 - UTC date and transfer completion time in ISO 8601 format.
                  - Upload id, if a resumable upload was performed.
-                 - Final result of the attempted transfer, success or failure.
+                 - Final result of the attempted transfer, either success or failure.
                  - Failure details, if any.
 
-                 If the log file already exists, gsutil will use the file as an
-                 input to the copy process, and will also append log items to
-                 the existing file. Files/objects that are marked in the
-                 existing log file as having been successfully copied (or
-                 skipped) will be ignored. Files/objects without entries will be
-                 copied and ones previously marked as unsuccessful will be
-                 retried. This can be used in conjunction with the -c option to
+                 If the log file already exists, gsutil uses the file as an
+                 input to the copy process, and appends log items to
+                 the existing file. Objects that are marked in the
+                 existing log file as having been successfully copied or
+                 skipped are ignored. Objects without entries are
+                 copied and ones previously marked as unsuccessful are
+                 retried. This option can be used in conjunction with the ``-c`` option to
                  build a script that copies a large number of objects reliably,
                  using a bash script like the following:
 
                    until gsutil cp -c -L cp.log -r ./dir gs://bucket; do
                      sleep 1
                    done
 
-                 The -c option will cause copying to continue after failures
-                 occur, and the -L option will allow gsutil to pick up where it
-                 left off without duplicating work. The loop will continue
-                 running as long as gsutil exits with a non-zero status (such a
-                 status indicates there was at least one failure during the
-                 gsutil run).
+                 The -c option enables copying to continue after failures
+                 occur, and the -L option allows gsutil to pick up where it
+                 left off without duplicating work. The loop continues
+                 running as long as gsutil exits with a non-zero status, which
```

Comment:
Recommend making this along the lines of "The loop continues running as long as gsutil exits with a non-zero status. A non-zero status indicates...: